src/PathautoManager.php

```diff
@@ -122,15 +128,31 @@ public function __construct(ConfigFactoryInterface $config_factory, ModuleHandle
   /**
    * {@inheritdoc}
    */
-  public function createAlias($module, $op, $source, $data, $type = NULL, $langcode = LanguageInterface::LANGCODE_NOT_SPECIFIED) {
-    $config = $this->configFactory->get('pathauto.settings');
-
+  public function createAlias(EntityInterface $entity, $op) {
```

Comment:
Maybe we should rename this method to createEntityAlias? it was entity specific before, even more so now. The whole manager pretty much is.